docs/VDTS_Setup.md

```diff
@@ -0,0 +1,25 @@
+# Setting up Virtual Device
+## Background
+We need a Virtual Device in order to test your Alexa skills. What is a Virtual Device (VD)? it is like an Amazon Echo but made with code.
+
+We'll create a complimentary VD for your, the only thing we need you to do is to allow this VD to access your Amazon Skills. This is done through the creation of an authorization token.
```

Comment:
"We'll create a complimentary virtual device for you"

Reply:
"the only thing we need you to do is to grant us to access to your Amazon Alexa account - and it is just your Alexa account. Once completed, you will have a Virtual Device token you can use in your tests."